test/PredictiveDeployer.admin.t.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+import { Test } from "forge-std/Test.sol";
+import { PredictiveDeployer } from "../src/PredictiveDeployer.sol";
+import { ERC1967Proxy } from "../src/dependencies/proxy/ERC1967Proxy.sol";
+import { IERC20 } from "../src/dependencies/token/interfaces/IERC20.sol";
+import { IPredictiveDeployerAdmin } from "../src/interfaces/IPredictiveDeployerAdmin.sol";
+import { CONTRACT_DEPLOYER, TEST_ERC20_TOKEN } from "./common/constants.t.sol";
+
+contract PredictiveDeployerTest is Test {
+    /* solhint-disable func-name-mixedcase */
```

Comment:
Consider disabling func-name-mixedcase for all test files in .solhint.json to prevent littering code with an excessive amount of "solhint-disable" lines.

Reply:
Do you know how do to this specifically for test files?